pages/platform/private-registry/responsibility-model/guide.en-gb.md

```diff
@@ -0,0 +1,176 @@
+| --- | --- | --- |
+| Monitor the proper functioning of physical devices (utilities) in support of the registry service |   | RA |
+| Monitor the proper functioning of virtual devices (utilities) in support of the registry service | I | RA |
+| Process and pay for alarms from managed devices in the Private Registry infrastructure | I | RA |
+| Maintain and monitor logs generated by the Private Registry service on the management infrastructure |  | RA |
+| Keep logs of the Private Registry service provided in Harbor UI or by using API | RA |  |
+
+##### 3.1.4. Storage
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Manage content hosted on the Private Registry service  | RA | I |
+| Manage data continuity and sustainability | RA | |
+| Perform storage and device maintenance |  | RA |
+
+##### 3.1.5. Connectivity
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Operate automatic network management systems (architecture, implementation, software and hardware maintenance for deployed public and private networks) | I | RA |
+| Provide and maintain a domain name on the service | I |  RA |
+
+##### 3.1.6. Management
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Provide inventory of the service used | I | RA |
+| Manage risks on managed components of the Private Regitry service |  | RA |
+| Maintain the service in operational and security conditions |  | RA |
+| Plan vulnerability scans on used impages (in case of subscription to M or L plan or with the client's proper tool)  | RA | I |
+| Manage alerts raised by vulnerability scans on images | RA |  |
+
+##### 3.1.7. Business continuity
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Manage automatic management systems and availability for the managed Platform service | I | RA |
+| Maintain a business continuity and disaster recovery plan on the Registry images | RA | I |
+
+#### 3.2. Event management
+
+##### 3.2.1. Incidents
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Notify incidents wihin the service Registry | RA | I |
+| Intervene with Public Cloud Private Registry managed elements | I | RA |
+
+##### 3.2.2. Changes
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Comply with the service Lifecycle Policy | RA | |
+| Deploy patches, update software and information systems hosted in Public Cloud Private Registry  | I | RA |
+| Deploy patches, update and configure the information system using Public Cloud Private Registry | RA |  |
+| Perform preventive interventions on managed elements of the Public Cloud Private Registry  | I | AR |
+
+### 4. Reverting
+
+#### 4.1. Reversibility model
+
+| **Activity** | **Customer** | **OVHcloud** |
+| --- | --- | --- |
+| Schedule reversibility operations | RA | I |
+| Choose fallback infrastructures | RA |  |
+| Use Docker to transfer image by image or replicate images by using Harbor registry | RA |  |
```

Comment:
```suggestion
| Use Docker to transfer image by image or replicated images by using Harbor registry | RA |  |
```

Reply:
could you correct it please ? 
Thank you

Reply:
It's been edited